ethercat_master:
  time_step:                          0.0025
  update_rate_too_low_warn_threshold: 50
  ros_namespace:                      ethercat_master

ethercat_devices:
  - type:               Nanotec
    name:               Nanotec
    configuration_file: device_configurations/nanotec.yaml
    ethercat_bus:       enx782d7e117ff5
    ethercat_address:   1

  # - type:               Maxon
  #   name:               Maxon
  #   configuration_file: device_configurations/maxon.yaml
  #   ethercat_bus:       enx782d7e117ff5
  #   ethercat_address:   1

  # - type:               Elmo
  #   name:               ElmoTwitter
  #   configuration_file: device_configurations/elmo.yaml
  #   ethercat_bus:       eth0
  #   ethercat_address:   2

  # - type:               Anydrive
  #   name:               Dynadrive
  #   configuration_file: device_configurations/anydrive.yaml
  #   ethercat_bus:       eth0
  #   ethercat_address:   3
  #   ethercat_pdo_type:  C

  # - type:               Rokubi
  #   name:               Rokubi
  #   configuration_file: device_configurations/rokubi.yaml
  #   ethercat_bus:       eth0
  #   ethercat_address:   4
  #   ethercat_pdo_type:  Z
